go.mod

```diff
@@ -81,5 +81,7 @@ replace (
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20190813201236-5a5508328169
 )
 
+replace github.com/RHsyseng/operator-utils => github.com/RHsyseng/operator-utils v0.0.0-20190904074031-4df1e0b603d2
```

Comment:
Please use the latest version

Reply:
Thanks. I have changed it to use the latest version.